src/gcp/rules.ts

```diff
@@ -57,17 +57,17 @@ function getLatestRulesetName(projectId, service) {
 
 /**
  * Gets the full contents of a ruleset.
- * @param {String} name Name of the ruleset.
+ * @param name Name of the ruleset.
  * @return {Array<Object>} Array of files in the ruleset. Each entry has form { content, name }.
  */
-function getRulesetContent(name) {
+export function getRulesetContent(name: string): Promise<object[]> {
```

Comment:
Is this object well known? Can we create a type rather than using `object`? (the docs above seem to say that they are...)

Reply:
done